examples/sklearn-kmeans/job_configs/sklearn_kmeans_base/app/custom/app_opt/sklearn/joblib_model_param_persistor.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from joblib import dump, load
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_constant import FLContextKey
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.abstract.model import (
+    ModelLearnable,
+    ModelLearnableKey,
+    make_model_learnable,
+)
+from nvflare.app_common.abstract.model_persistor import ModelPersistor
+from nvflare.app_common.app_constant import AppConstants
+
+
+class JoblibModelParamPersistor(ModelPersistor):
```

Comment:
I think ModelPersistor is an abstract for "Deep Learning Models"

In our case here, could you try to derive from "LearnablePersistor" "https://github.com/NVIDIA/NVFlare/blob/dev/nvflare/app_common/abstract/learnable_persistor.py#L22"

Reply:
changed